src/objects/Gate.tscn

```diff
@@ -1,20 +1,37 @@
-[gd_scene load_steps=5 format=2]
+[gd_scene load_steps=12 format=2]
 
-[ext_resource path="res://assets/objects/open_gate.png" type="Texture" id=1]
-[ext_resource path="res://assets/objects/closed_gate.png" type="Texture" id=2]
+[ext_resource path="res://assets/objects/gate_0.png" type="Texture" id=1]
 [ext_resource path="res://src/objects/Gate.gd" type="Script" id=3]
+[ext_resource path="res://assets/objects/gate_1.png" type="Texture" id=4]
+[ext_resource path="res://assets/objects/gate_2.png" type="Texture" id=5]
+[ext_resource path="res://assets/objects/gate_3.png" type="Texture" id=6]
+[ext_resource path="res://assets/objects/gate_4.png" type="Texture" id=7]
+[ext_resource path="res://assets/objects/gate_7.png" type="Texture" id=8]
+[ext_resource path="res://assets/objects/gate_6.png" type="Texture" id=9]
+[ext_resource path="res://assets/objects/gate_5.png" type="Texture" id=10]
+[ext_resource path="res://assets/objects/gate_8.png" type="Texture" id=11]
 
 [sub_resource type="SpriteFrames" id=1]
 animations = [ {
-"frames": [ ExtResource( 1 ) ],
+"frames": [ ExtResource( 11 ), ExtResource( 8 ), ExtResource( 9 ), ExtResource( 10 ), ExtResource( 7 ), ExtResource( 6 ), ExtResource( 5 ), ExtResource( 4 ), ExtResource( 1 ) ],
+"loop": false,
+"name": "close",
+"speed": 20.0
```

Comment:
We don't need to worry about it for this pull request (or maybe even ever if we think it looks ok), but maybe we could make it so the animation speed option also increases the speed of the open/close animations